Yoast/wordpress-seo
# Change configuration wizard for plugin training

## Summary
1. Removed the current video in the last step of the config wizard and added a screenshot of the Yoast SEO plugin training, linking to https://academy.yoast.com/courses/yoast-seo-wordpress-2018/. 
2. Removed the Yoast SEO plugin training from step 8.
3. Combined step 7 and 8:
    * Title became 'Continue learning'
    * The "keyword research training" and (in free) the "upgrade to premium" suggestions are placed under the newsletter signup.
4. Changed the copy of the Success page (step 8) to:

> Yoast SEO will now take care of all the needed technical optimization of your site. To really improve your site's performance in the search results, it's important to know everything our plugin has to offer. Sign up for our free Yoast SEO plugin training, in which you'll learn how to use Yoast SEO and how it can help you make the best of your website!


This PR can be summarized in the following changelog entry:

* Enhances the configurations wizard's content and combines step 7 and 8

## Relevant technical choices:

* Added the suggestions component in the newsletter field

## Test instructions

This PR can be tested by following these steps:

* Go to SEO -> General and open the configuration wizard.
* Go to step 7 and check if everything is displayed correctly.
    * Check if both the newsletter signup component and the suggestions show up correctly
* Go to step 8 and check if everything is displayed correctly.
    * Check if the focus of the Yoast_Academy_video.png image works as expected.
    * Check if the link to the course is accessible.
    * Check if the Yoast_Academy_video.png image is loaded correctly
    * Check if the Short link and UTM tags on the image work as expected.

## UI changes

## Documentation

## Quality assurance


Fixes #https://github.com/Yoast/wordpress-seo-premium/issues/2542
